contracts/governance/compatibility/GovernorCompatibilityBravo.sol

```diff
@@ -99,20 +99,15 @@ abstract contract GovernorCompatibilityBravo is IGovernorTimelock, IGovernorComp
         );
     }
 
-    function cancel(uint256 proposalId) public virtual override {
+    function cancel(uint256 proposalId) public virtual override(IGovernor, Governor) {
```

Comment:
Needs docs explaining the logic, also that it overrides and doesn't call `super.cancel` from the core Governor.